src/internal/analytics/__integ__/static-multi-page-create.test.ts

```diff
@@ -0,0 +1,341 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import { BasePageObject } from '@cloudscape-design/browser-test-tools/page-objects';
+import useBrowser from '@cloudscape-design/browser-test-tools/use-browser';
+import createWrapper from '../../../../lib/components/test-utils/selectors';
+
+interface ExtendedWindow extends Window {
+  __awsuiFunnelMetrics__: Array<any>;
+}
+
+declare const window: ExtendedWindow;
+
+const wrapper = createWrapper();
+const FUNNEL_INTERACTION_ID = 'mocked-funnel-id';
+
+class MultiPageCreate extends BasePageObject {
+  async getFunnelLog() {
+    const funnelLog = await this.browser.execute(() => window.__awsuiFunnelMetrics__);
+    const actions = funnelLog.map(item => item.action);
+    return { funnelLog, actions };
+  }
+}
+
+const setupTest = (testFn: (page: MultiPageCreate) => Promise<void>) => {
+  return useBrowser(async browser => {
+    const page = new MultiPageCreate(browser);
+    await browser.url('#/light/funnel-analytics/static-multi-page-flow');
+    await testFn(page);
+  });
+};
+
+describe('Multi-page create', () => {
+  test(
+    'Starts funnel and funnel step and page is loaded',
+    setupTest(async page => {
+      const { funnelLog, actions } = await page.getFunnelLog();
+      expect(actions).toEqual(['funnelStart', 'funnelStepStart']);
+
+      const [funnelStartEvent, funnelStartStepEvent] = funnelLog;
```

Comment:
minor: we could rename "funnelStartStepEvent" to "funnelStepStartEvent" to align with the event name

Reply:
Will follow up with a fix.